docs/guides/serverless-hamsim-template.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<blockquote>By now you've seen a basic example of how to get started writing, uploading, and running a program with Qiskit Serverless. The workloads you are building for your own use cases are likely not so simple. For example, you might need to carefully consider what domain-level inputs and outputs are relevant for your application, how to make sure your program can be reusable across a range of those inputs, and what kind of information you need returned to you <em>during</em> the execution of your job so that you can better evaluate the progress of your workload. You may even want to incorporate a dry run mode that lets you test the impact of a set of particular inputs before sending the resulting circuits to hardware.</blockquote><blockquote><br></blockquote><blockquote>A function template is an example of a realistic workload using a specific application domain to contextualize these aspects. It is meant to be a starting point for you to take and modify for your own needs so that you don't have to start from scratch.</blockquote><blockquote><br></blockquote><blockquote>Here, we will see a function template for addressing Hamiltonian simulation problems.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>